react-frontend/src/login/LoginImageComponent.test.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { render } from "@testing-library/react";
+
+import LoginImageComponent from "login/LoginImageComponent";
+import { IMG_BASE_PATH } from "constants/appConstants";
+import Image from "../../public/assets/images/cat-img.png";
```

Comment:
@avi4630 why do we need this?

Reply:
@sahilbhatia for test case of props.when we give size="lg" then render different image therefore
we use this path to check image render correctly or not